src/Library/DeviceBase.cpp

```diff
@@ -90,14 +90,20 @@ namespace usbguard
      * Set connect type
      */
     int retry = 0;
-    std::string connect_type;
-    while (retry < 20) {
+    std::string connect_type = "";
+    /*
+     * Read port/connect_type only for non Host Controller devices
+     */
+    if (getPort().substr(0, 3) != "usb") {
+      while (retry < 10) {
         connect_type = sysfs_device.readAttribute("port/connect_type", /*strip_last_null=*/true, /*optional=*/true);
         if (connect_type != "") {
             break;
         }
         USBGUARD_LOG(Trace) << "connect_type is empty, will attempt again";
+        usleep(200);
         retry++;
+      }
```

Comment:
Please note that on target branch `master` formatting of file `src/Library/DeviceBase.cpp` has changed in the mean time (in commit be5d74bd34458b9c02ff352599b73a34417e3c3e) and we also gained additional CI in the mean time that should get a say in this PR prior to a merge. Please rebase your PR onto latest `master`, fix the unavoidable conflicts, use script `./scripts/reformat-sources.sh` to ensure formatting that the CI will have a chance to like, and then force-push back into the same pull request. Thank you!